### PR TITLE
Repo cleanup Feb 21

### LINKED ISSUE #22375: Repo chores - February
- [x] Check broken links report for [dotnet/docs](https://ops.microsoft.com/#/repos/a3bda507-3390-de91-8f7f-26f90f4e5fc8?tabName=brokenlinks)
- [x] Check broken links report for [dotnet/dotnet-api-docs](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks)
- [x] Check [microsoftdocs/feedback](https://github.com/microsoftdocs/feedback/issues) for .NET issues
- [x] Run CleanRepo and check for orphaned articles
- [x] Run CleanRepo and delete orphaned images
- [x] Run CleanRepo and fix up links (redirects, site-relative)

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #22375. Deletes orphaned images. Adds orphaned article to TOC.

### CODE REVIEW

File: docs/architecture/porting-existing-aspnet-apps/logging-differences.md [hunk 1]
```diff
@@ -1,54 +0,0 @@
----
```
Comment (Youssef1313) [inline docs/architecture/porting-existing-aspnet-apps/logging-differences.md:1]: I don't see a redirection for this one. Is that intentional?

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

No warnings on the build, so all these are orphaned.

File: docs/architecture/porting-existing-aspnet-apps/logging-differences.md (hunk 1 above)
Comment (gewarren) [reply docs/architecture/porting-existing-aspnet-apps/logging-differences.md:1]: @ardalis @IEvangelist This file wasn't added to the TOC when it was added on Feb 11 in https://github.com/dotnet/docs/pull/22167. Should it be deleted or added to the TOC?

Comment (IEvangelist) [reply docs/architecture/porting-existing-aspnet-apps/logging-differences.md:1]: Yes, I believe that should be added to the TOC, thanks for catching that!

**Pull request merged**